client/js/util/misc.js

```diff
@@ -211,6 +211,15 @@ function getPrettyName(tag) {
     return tag;
 }
 
+function preloadPostImages(post) {
+    if (!["image", "animation"].includes(post.type)) {
+        return;
+    }
+    const img = new Image()
```

Comment:
Nit: missing `;`

Reply:
Code formatters were added in #329 but to my knowledge were never run again. I think a lot of the codebase still has inconsistencies. I'll of course fix it here to not introduce even more, but important to keep that in mind if you want to re-format everything (after dealing with PRs as to not create conflicts everywhere).